sycl/test/on-device/basic_tests/interop/traits.cpp

```diff
@@ -0,0 +1,59 @@
+// RUN: %clangxx -fsycl -DUSE_OPENCL %s
```

Comment:
What causes the force of the backend at run-time?

Reply:
It's a compile-only test, the program is not really run